apisix/plugins/jwt-auth.lua

```diff
@@ -48,7 +48,12 @@ local schema = {
         hide_credentials = {
             type = "boolean",
             default = false
-        }
+        },
+        key_claim_name = {
+            type = "string",
+            default = "key",
+            minLength = "1",
```

Comment:
this wrong, we should use `minLength = 1`, it should be a number

https://json-schema.org/understanding-json-schema/reference/string

<img width="876" alt="image" src="https://github.com/user-attachments/assets/e4c7a0d9-b837-4e05-a350-6ccd0dc894c4">

Reply:
fixed